packages/dapi/test/unit/transactionsFilter/getHistoricalBlockHeadersIteratorFactory.spec.js

```diff
@@ -47,7 +48,8 @@ describe('getHistoricalBlockHeadersIteratorFactory', () => {
     expect(r3.done).to.be.false();
     expect(r4.done).to.be.true();
 
-    expect(coreRpcMock.getBlock.callCount).to.be.equal(1);
+    expect(coreRpcMock.getBlock.callCount).to.be.equal(0);
```

Comment:
I think we probably don't need this line anymore? 
In the other places as well.